draft-ietf-quic-reliable-stream-reset.md

```diff
@@ -133,6 +135,31 @@ CLOSE_STREAM frames are ack-eliciting. When lost, they MUST be retransmitted,
 unless the stream state has transitioned to "Data Recvd" or "Reset Recvd" due
 to transmission and acknowledgement of other frames (see {{multiple-frames}}).
 
+## ENOUGH Frame
+
+Conceptually, the CLOSE_STREAM frame is a STOP_SENDING frame with an
+added Reliable Size field.
+
+ENOUGH Frame {
+  Type (i) = TBD,
+  Stream ID (i),
+  Application Protocol Error Code (i),
+  Offset (i),
```

Comment:
```suggestion
  Reliable Size (i),
```
We avoided the term "offset" in RFC 9000 (https://github.com/quicwg/base-drafts/issues/2262), my preference would be to not use that here.

Reply:
Done.